src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -746,6 +751,35 @@ class ConsentGroupReview extends Component {
     });
   };
 
+  shouldShowThis = (label) => {
```

Comment:
I am not a fan of putting this logic here. If it's **just** here in UI code, then the APIs will never know about it. Not sure how much work it would be, but the objects being returned via the API should also know if a thing is editable or not. 

See also the duplicate code in ProjectReview.js. If we do keep this purely UI, then it should be a utility function instead.

Reply:
Agree. Ill review backend behavior in all cases.